src/components/heading/heading.stories.mdx

```diff
@@ -42,6 +42,18 @@ import Heading from "carbon-react/lib/components/heading";
   </Story>
 </Canvas>
 
+### Heading with headingType
+
+The `headingType` prop changes the HTML heading element of the `title` string which is displayed within the component. Each `string` 
+represents a HTML Section Heading element e.g `h1` is equal to `<h1>`, allowing up to six sections of headings from most to least important.
+
```

Comment:
I'm not a writer - this might be worth asking our technical authors about! - but there are a few things about this paragraph that I don't like:

- I'm not sure why `string` is in code formatting. Anyway I'd personally say "each option" rather than "each string".
- "HTML Section Heading element" reads weird to me, particular with the words capitalised which make it sound like an "official" term for something. It's just an HTML heading element so I'd say that.
- "`h1` is equal to `<h1>`" - I know what you mean, and readers likely will too, but this is clumsy: the two things are not "equal".
- I really don't think we need to explain about the HTML heading hierarchy, consumers should be expected to know this, and even if they somehow don't I don't think this brief sentence adds much to our documentation
- I think "an HTML" is more correct than "a HTML" (at least it matches what I say, but I recognise this is potentially controversial so don't worry about it if you much prefer "a")

I would replace all of the above with something like:

```
The `headingType` prop sets the HTML heading element which is rendered on the page. It can take values `"h1"` up to `"h6"`.
```

or words roughly to that effect, but as I said I'm no writer so feel free to make changes or even rewrite it yourself to take account of some of my suggestions (it doesn't have to be all if you don't agree with them!).